lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -131,7 +131,15 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                 % if is_unfulfilled_entitlement:
                     <span class="info-date-block" aria-live="polite">
                         <span class="icon fa fa-warning" aria-hidden="true"></span>
-                        ${_('You must select a session to access the course.')}
+                        % if entitlement_expired_at:
+                            ${_('You can no longer select a session, your final day to select a session was {entitlement_expired_at}.').format(entitlement_expired_at=entitlement_expired_at)}
```

Comment:
@HarryRein Just a minor punctuation tweak: "You can no longer select a session. Your final day to select a session was..."

Reply:
Added